.travis.yml

```diff
@@ -31,6 +34,7 @@ matrix:
 before_install:
   - echo "memory_limit=4G" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
   - phpenv config-rm xdebug.ini || true
+  - composer global require --no-progress --no-scripts --no-plugins symfony/flex dev-master
```

Comment:
I am not sure but if we use Flex, maybe we can also use the constraint `extra.symfony.version` in place `SYMFONY_REQUIRE` ? And edit composer.json on each stage with `jq`

Reply:
Or you simply add `composer config extra.symfony.require $SYMFONY_REQUIRE` instead of introducing flex to travis.